### PR TITLE
ci: fix lerna publish

### PR DESCRIPTION
**Pull request by Yeti-or**
:baby_chick: Download canary assets:

[color_sberHealth_ios-swift--canary.534.5133965229.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.534.5133965229.swift)  
[color_sberHealth_kotlin--canary.534.5133965229.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.534.5133965229.kt)  
[color_sberHealth_react-native--canary.534.5133965229.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.534.5133965229.ts)  
[color_sbermarket_business_ios-swift--canary.534.5133965229.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.534.5133965229.swift)  
[color_sbermarket_business_kotlin--canary.534.5133965229.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.534.5133965229.kt)  
[color_sbermarket_business_react-native--canary.534.5133965229.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.534.5133965229.ts)  
[color_sbermarket_ios-swift--canary.534.5133965229.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.534.5133965229.swift)  
[color_sbermarket_kotlin--canary.534.5133965229.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.534.5133965229.kt)  
[color_sbermarket_metro_ios-swift--canary.534.5133965229.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.534.5133965229.swift)  
[color_sbermarket_metro_kotlin--canary.534.5133965229.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.534.5133965229.kt)  
[color_sbermarket_metro_react-native--canary.534.5133965229.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.534.5133965229.ts)  
[color_sbermarket_react-native--canary.534.5133965229.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.534.5133965229.ts)  
[color_sbermarket_selgros_ios-swift--canary.534.5133965229.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.534.5133965229.swift)  
[color_sbermarket_selgros_kotlin--canary.534.5133965229.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.534.5133965229.kt)  
[color_sbermarket_selgros_react-native--canary.534.5133965229.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.534.5133965229.ts)  
[color_sbermarket_wlbusiness_ios-swift--canary.534.5133965229.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_ios-swift--canary.534.5133965229.swift)  
[color_sbermarket_wlbusiness_kotlin--canary.534.5133965229.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_kotlin--canary.534.5133965229.kt)  
[color_sbermarket_wlbusiness_react-native--canary.534.5133965229.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_react-native--canary.534.5133965229.ts)  
[color_sberonline_ios-swift--canary.534.5133965229.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_ios-swift--canary.534.5133965229.swift)  
[color_sberonline_kotlin--canary.534.5133965229.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_kotlin--canary.534.5133965229.kt)  
[color_sberonline_react-native--canary.534.5133965229.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_react-native--canary.534.5133965229.ts)  
[color_sberprime_ios-swift--canary.534.5133965229.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.534.5133965229.swift)  
[color_sberprime_kotlin--canary.534.5133965229.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.534.5133965229.kt)  
[color_sberprime_react-native--canary.534.5133965229.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.534.5133965229.ts)  
[shadow_sbermarket_react-native--canary.534.5133965229.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.534.5133965229.ts)  
[typo_mage_ios-swift--canary.534.5133965229.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.534.5133965229.swift)  
[typo_mage_kotlin--canary.534.5133965229.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.534.5133965229.kt)  
[typo_mage_react-native--canary.534.5133965229.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.534.5133965229.ts)  
[typo_plasma_ios-swift--canary.534.5133965229.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.534.5133965229.swift)  
[typo_plasma_kotlin--canary.534.5133965229.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.534.5133965229.kt)  
[typo_plasma_react-native--canary.534.5133965229.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.534.5133965229.ts)  
[typo_ruler_ios-swift--canary.534.5133965229.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.534.5133965229.swift)  
[typo_ruler_kotlin--canary.534.5133965229.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.534.5133965229.kt)  
[typo_ruler_react-native--canary.534.5133965229.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.534.5133965229.ts)  
[typo_sage_ios-swift--canary.534.5133965229.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.534.5133965229.swift)  
[typo_sage_kotlin--canary.534.5133965229.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.534.5133965229.kt)  
[typo_sage_react-native--canary.534.5133965229.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.534.5133965229.ts)  
[typo_sbermarket_ios-swift--canary.534.5133965229.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.534.5133965229.swift)  
[typo_sbermarket_kotlin--canary.534.5133965229.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.534.5133965229.kt)  
[typo_sbermarket_react-native--canary.534.5133965229.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.534.5133965229.ts)  
[typo_soulmate_ios-swift--canary.534.5133965229.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.534.5133965229.swift)  
[typo_soulmate_kotlin--canary.534.5133965229.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.534.5133965229.kt)  
[typo_soulmate_react-native--canary.534.5133965229.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.534.5133965229.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.215.1-canary.534.5133965229.0
  npm install @salutejs/plasma-core@1.120.1-canary.534.5133965229.0
  npm install @salutejs/plasma-hope@1.215.1-canary.534.5133965229.0
  npm install @salutejs/plasma-icons@1.149.1-canary.534.5133965229.0
  npm install @salutejs/plasma-temple@1.164.1-canary.534.5133965229.0
  npm install @salutejs/plasma-tokens@1.54.1-canary.534.5133965229.0
  npm install @salutejs/plasma-typo@0.33.1-canary.534.5133965229.0
  npm install @salutejs/plasma-ui@1.196.1-canary.534.5133965229.0
  npm install @salutejs/plasma-web@1.215.1-canary.534.5133965229.0
  npm install @salutejs/plasma-cy-utils@0.61.1-canary.534.5133965229.0
  npm install @salutejs/plasma-sb-utils@0.118.1-canary.534.5133965229.0
  # or 
  yarn add @salutejs/plasma-b2c@1.215.1-canary.534.5133965229.0
  yarn add @salutejs/plasma-core@1.120.1-canary.534.5133965229.0
  yarn add @salutejs/plasma-hope@1.215.1-canary.534.5133965229.0
  yarn add @salutejs/plasma-icons@1.149.1-canary.534.5133965229.0
  yarn add @salutejs/plasma-temple@1.164.1-canary.534.5133965229.0
  yarn add @salutejs/plasma-tokens@1.54.1-canary.534.5133965229.0
  yarn add @salutejs/plasma-typo@0.33.1-canary.534.5133965229.0
  yarn add @salutejs/plasma-ui@1.196.1-canary.534.5133965229.0
  yarn add @salutejs/plasma-web@1.215.1-canary.534.5133965229.0
  yarn add @salutejs/plasma-cy-utils@0.61.1-canary.534.5133965229.0
  yarn add @salutejs/plasma-sb-utils@0.118.1-canary.534.5133965229.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-534/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-534/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-534/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-534/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-534/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-534/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-534/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Review by neretin-trike — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-534/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-534/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-534/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-534/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-534/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-534/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-534/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-534/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-534/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-534/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-534/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-534/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

File: packages/plasma-core/src/components/Spinner/Spinner.component-test.tsx [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import React from 'react';
 import { mount, CypressTestDecorator, getComponent, PadMe } from '@salutejs/plasma-cy-utils';
 import styled from 'styled-components';
-import { critical } from '@salutejs/plasma-tokens';
+
+// import { critical } from '@salutejs/plasma-tokens';
+const critical = 'var(--plasma-colors-critical, #E31227)';
```
Comment (Yakutoc) [inline packages/plasma-core/src/components/Spinner/Spinner.component-test.tsx:6]: @Yeti-or Я так понимаю это временное решение? Или для этого теста все равно что тут значение константа?

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

File: cypress/webpack.config.js [hunk 2]
```diff
@@ -49,12 +47,6 @@ module.exports = function getWebpackConfig() {
                 react: resolveInsidePackage('react'),
                 'react-dom': resolveInsidePackage('react-dom'),
                 '@salutejs/plasma-icons': resolveInsidePackage('@salutejs', 'plasma-icons'),
-                '@salutejs/plasma-cy-utils': resolveFromRoot('@salutejs', 'plasma-cy-utils'),
```
Comment (Yakutoc) [inline cypress/webpack.config.js:52]: @Yeti-or А можешь в двух словах объяснить почему оно теперь работает?

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-534/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-534/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-534/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-534/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-534/temple-storybook/

**Review by Yakutoc — APPROVED** [review]

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Review by kayman233 — APPROVED** [review]

**Review by neretin-trike — APPROVED** [review]

File: packages/plasma-core/src/components/Skeleton/Skeleton.component-test.tsx [hunk 3]
```diff
@@ -1,11 +1,29 @@
 import React from 'react';
-import styled from 'styled-components';
 import { mount, CypressTestDecorator, getComponent, PadMe, SpaceMe, withNoAnimation } from '@salutejs/plasma-cy-utils';
-import { typography } from '@salutejs/plasma-tokens';
 
 import { radiuses } from '../..';
 
-const textSizes = Object.keys(typography);
+// from  typography in '@salutejs/plasma-tokens'
```
Comment (neretin-trike) [inline packages/plasma-core/src/components/Skeleton/Skeleton.component-test.tsx:6]: можешь пожалута вынести это в ишью и прикрепить тут ссылочку в TODO?

File: packages/plasma-core/src/components/Toast/ToastBase.component-test.tsx [hunk 4]
```diff
@@ -1,7 +1,9 @@
 import React from 'react';
 import { mount, CypressTestDecorator, getComponent } from '@salutejs/plasma-cy-utils';
 import { IconClose } from '@salutejs/plasma-icons';
-import { critical } from '@salutejs/plasma-tokens-b2c';
+
+// import { critical } from '@salutejs/plasma-tokens-b2c';
```
Comment (neretin-trike) [inline packages/plasma-core/src/components/Toast/ToastBase.component-test.tsx:5]: тут тоже

File: packages/plasma-core/src/components/Spinner/Spinner.component-test.tsx (hunk 1 above)
Comment (Yeti-or) [reply packages/plasma-core/src/components/Spinner/Spinner.component-test.tsx:6]: В целом для теста без разницы какой цвет будет передан на самом деле

File: cypress/webpack.config.js (hunk 2 above)
Comment (Yeti-or) [reply cypress/webpack.config.js:52]: Потому что в каждой из библиотек стоит зависимость `'@salutejs/plasma-cy-utils'`
web
b2c 
ui 
и core в которой в том числе лежат тесты, и насколько я понял проблему: библиотеки резолвятся относительно этих тестов, но на самом деле я не залезал в кишки cypress и webpack чтобы точно все отдебажить, тем более я не помню почему было принято решение тут резолвить все эти модули;
Помню что во времена моей молодости cypress open и cypress run вели себя по разному и с разными ошибками падали, тогда-то и были внедрены эти костыли, но как видишь никакой документации привычки писать тогда не было, как впрочем и сейчас, такова боль разработки и оставляем мы её для будущих нас, которые будут с недовольством и недоумением вычищать оставленные нами паллиативы.

File: packages/plasma-core/src/components/Skeleton/Skeleton.component-test.tsx (hunk 3 above)
Comment (Yeti-or) [reply packages/plasma-core/src/components/Skeleton/Skeleton.component-test.tsx:6]: могу но в отдельном пр

**Pull request merged**